### PR TITLE
fix: setup release please

### PR DESCRIPTION
**Pull request by JoeWang1127**

### CODE REVIEW

**Review by suztomo — APPROVED** [review]

**Comment by suztomo** [comment]
https://github.com/GoogleCloudPlatform/spring-cloud-gcp/blob/main/.github/release-trigger.yml

Search for any file that has "release" in spring-cloud-gcp.

Search for any file that has "release-please" in spring-cloud-gcp.

You need the version.txt.

**Comment by JoeWang1127** [comment]
> Search for any file whose name has "release" in spring-cloud-gcp.

> Search for any file that has "release-please" in spring-cloud-gcp.

Only two files have "release" in name: `release-please.yml` and `release-trigger.yml`.

**Review by suztomo — APPROVED** [review]

File: app-maven-plugin/pom.xml [hunk 1]
```diff
@@ -4,7 +4,7 @@
```
Comment (suztomo) [inline app-maven-plugin/pom.xml:4]: This module depends on appengine-plugins-core. Release Please should update that line too.

```
  <dependencies>
    <dependency>
      <groupId>com.google.cloud.tools</groupId>
      <artifactId>appengine-plugins-core</artifactId>
      <version>0.10.1-SNAPSHOT</version>
```

**Comment by suztomo** [comment]
Release Please and Release Trigger are installed in this repository. Good.

Screenshot 2023-12-12 at 8 03 58 PM

File: app-maven-plugin/pom.xml (hunk 1 above)
Comment (JoeWang1127) [reply app-maven-plugin/pom.xml:4]: Ah, good catch, thanks.

**Comment by suztomo** [comment]
When merging this pull request, update the title (+commit message) with "fix:" prefix so that Release Please should create a pull request.

**Comment by JoeWang1127** [comment]
> When merging this pull request, update the title (+commit message) with "fix:" prefix so that Release Please should create a pull request.

Done.

**Review by suztomo — APPROVED** [review]
Let's merge this.

**Pull request merged**